cove_iati/management/commands/iati_cli.py

```diff
@@ -17,19 +16,28 @@ def add_arguments(self, parser):
                             'Org-ids prefixes')
         super(Command, self).add_arguments(parser)
 
-    def handle(self, file, *args, **options):
-        super(Command, self).handle(file, *args, **options)
+    def handle(self, files, *args, **options):
+        super(Command, self).handle(files, *args, **options)
+        stream = options.get('stream')
         openag = options.get('openag')
         orgids = options.get('orgids')
 
         try:
-            result = iati_json_output(self.output_dir, file, openag=openag, orgids=orgids)
+            if stream:
+                for file in files:
+                    if os.path.isdir(file):
+                        self.stdout.write('Skipping %s directory ' % str(file))
+                    else:
+                        result = iati_json_output(self.output_dir, file, openag=openag,
+                                                  orgids=orgids)
+                        self.stdout.write(str(result))
+            else:
+                result = iati_json_output(self.output_dir, files[0], openag=openag, orgids=orgids)
+                with open(os.path.join(self.output_dir, "results.json"), 'w+') as result_file:
+                    json.dump(result, result_file, indent=2, cls=SetEncoder)
         except APIException as e:
             self.stdout.write(str(e))
             sys.exit(1)
         except CoveInputDataError as e:
             self.stdout.write('Not well formed XML: %s' % str(e.context['error']))
```

Comment:
stdout also.

We should probably move both these exceptions up to wrap ```iati_json_output(..)```
As if one file fails we probably want a way for the streaming to continue.
We could have a special json object in the stream like
```{"file": "/path/to/file", "error": "what the error is"}```